ush/python/pyufs/ufswm/gfs.py

```diff
@@ -0,0 +1,70 @@
+"""
```

Comment:
I don't agree w/ the way the class hierarchy is being setup and defined.

Reply:
I thought this had been resolved during the Google Hangout with Walter Kolczynski. You had stated that you liked how GFS was a sub-class of UFSWM hence the defined hierarchy.

Reply:
May be I wasn't clear.
We need a `forecast.py` task that is a generic forecast task.  It contains a `class Forecast(Task)`. IMO, we don't really need it, but if you want to introduce that separation, this would be it.

The `gfs_forecast.py` contains a `class GFSForecast(Forecast, GFS)`

`GFS` is a class from `gfs.py` that contains the methods and definitions specific to the global configuration of the  ufs-weather-model.  It is derived from `ufs.py`, such that `class GFS(UFS)`

Reply:
Is that not what I have done here: https://github.com/HenryWinterbottom-NOAA/global-workflow/blob/feature/gw_atm_forecast_init/ush/python/pyufs/ufswm/ufswm.py#L43

and here: https://github.com/HenryWinterbottom-NOAA/global-workflow/blob/feature/gw_atm_forecast_init/ush/python/pyufs/ufswm/gfs.py#L41

What am I not understanding? Is this a naming issue or is there something more fundamental?